### PR TITLE
outplacement: AFC-1116

### PR DESCRIPTION
**Pull request by nils-vertel**

### CODE REVIEW

File: outplacement/models/outplacement.py [hunk 1]
```diff
@@ -108,10 +104,10 @@ def _read_group_stage_ids(self, stages, domain, order):
         return stages.search([], order=order)
 
     @api.model
-    def _read_group_department_ids(self, departments, domain, order):
+    def _read_group_performing_operation_ids(self, performing_operation, domain, order):
         """ Always display all stages """
-        _logger.warn('group by department domain %s order %s' % (domain,order))
-        return departments.search([], order=order)
+        _logger.warn('group by performing_operation_ids %s order %s' % (domain,order))
```
Comment (nils-vertel) [inline outplacement/models/outplacement.py:109]: Warn seems to be quite high logging level, consider choosing a lower (INFO?)

File: outplacement/models/outplacement.py [hunk 2]
```diff
@@ -128,20 +124,19 @@ def write(self, vals):
...
 
     @api.multi
-    def _compute_my_department(self):
-        department = self.env['hr.employee'].search(
-            [('user_id', '=', self.env.user.id)]).mapped('department_id')
-        department = department[0] if len(department) > 0 else None
-        return self.filtered(lambda o: o.department_id == department)
+    def _compute_my_performing_operation(self):
+        return self.env.user.performing_operation_ids
+        # ~ performing_operation_ids  = elf.env.user.ids
```
Comment (nils-vertel) [inline outplacement/models/outplacement.py:129]: Consider removing this comments

File: outplacement/models/outplacement.py [hunk 3]
```diff
@@ -27,12 +27,12 @@ def _default_stage_id(self):
 
     @api.model
     def _read_group_employee_ids(self, employees, domain, order):
-        """ Always display all stages """
+        """ Always display all employees in users performing_operation """
         _logger.warn('group by employee domain %s order %s' % (domain, order))
```
Comment (nils-vertel) [inline outplacement/models/outplacement.py:31]: Consider changing log level to INFO, so its possible to suppress when we don't want to see it

**Review by nils-vertel — COMMENTED** [review]
Crashes when:

- Simulate suborder from BAR
- Using filter "My Outplacements"

File: outplacement_crmsync/models/res_partner.py [hunk 4]
```diff
@@ -195,16 +195,24 @@ def get_af_management_team(self):
         partner = xmlrpc.common.env['res.partner'].browse(
             xmlrpc.common.env['res.partner'].search([('email','=',self.management_team_id.email)],limit=1))
             # ~ xmlrpc.common.env['res.partner'].search([('id','=',26)],limit=1))
```
Comment (arturchilangwa-vertel) [inline outplacement_crmsync/models/res_partner.py:197]: Remove commented code

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

**Comment by anderswallenquist** [comment]
Looks good

**Pull request merged**